ConceptMap/ConceptMap-Genomics-type-of-transplant-procedure.json

```diff
@@ -0,0 +1,481 @@
+              "equivalence": "equivalent"
+            },
+            {
+              "code": "18027006",
+              "display": "Transplantation of liver (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ocular-tissue",
+          "display": "OCULAR TISSUE",
+          "target": [
+            {
+              "code": "120095006",
+              "display": "Eye structure transplantation (procedure)",
+              "equivalence": "wider"
+            }
+          ]
+        },
+        {
+          "code": "pancreas",
+          "display": "PANCREAS ONLY",
+          "target": [
+            {
+              "code": "62438007",
+              "display": "Transplantation of pancreas (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "pancreas-and-liver",
+          "display": "PANCREAS AND LIVER",
+          "target": [
+            {
+              "code": "62438007",
+              "display": "Transplantation of pancreas (procedure)",
+              "equivalence": "equivalent"
+            },
+            {
+              "code": "62438007",
+              "display": "Transplantation of pancreas (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "pancreas-islets",
+          "display": "PANCREAS ISLETS",
+          "target": [
+            {
+              "code": "128531006",
+              "display": "Islet cell transplant (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "pancreas-kidney",
+          "display": "PANCREAS KIDNEY",
+          "target": [
+            {
+              "code": "6471000179103",
+              "display": "Transplantation of kidney and pancreas (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "pancreas-liver-and-kidney",
+          "display": "PANCREAS LIVER AND KIDNEY",
+          "target": [
+            {
+              "code": "62438007",
+              "display": "Transplantation of pancreas (procedure)",
+              "equivalence": "equivalent"
+            },
+            {
+              "code": "62438007",
+              "display": "Transplantation of pancreas (procedure)",
+              "equivalence": "equivalent"
+            },
+            {
+              "code": "70536003",
+              "display": "Transplant of kidney (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "partial-lung",
+          "display": "PARTIAL LUNG",
+          "target": [
+            {
+              "code": "232657004",
+              "display": "Single lung transplant (procedure)",
+              "equivalence": "equivalent"
```

Comment:
This might be wider, use 429332008 | Transplantation of single lobe of lung (procedure) | instead (and state equivalence narrower)

Reply:
Kept relationship as equivalent by virtue of this definition of partial-lung.

A partial lung transplant, also known as a lobar-lung transplant, is a procedure that replaces a diseased portion of a lung with a healthy lobe from a donor. It can be an option for people who are small or have space issues, or for those who are waiting for an organ